receiver/README.md

```diff
@@ -15,32 +15,70 @@ Supported receivers (sorted alphabetically):
 - [Zipkin Receiver](#zipkin)
 
 ## Configuring Receiver(s)
-TODO - Add what a fullname is and how that is referenced in other parts of the
-configuration. Describe the common receiver settings: endpoint, disabled and
-disable-backpressure.
 
-## <a name="opencensus"></a>OpenCensus Receiver
-**Traces and metrics are supported.**
+Receivers are configured via YAML under the top-level `receivers` tag. There
+must be at least one enabled receiver for this configuration to be considered
+valid.
 
-This receiver receives spans from [OpenCensus](https://opencensus.io/) instrumented applications and translates them into the internal format sent to processors and exporters in the pipeline.
+The following is a sample configuration for the `examplereceiver`.
+```yaml
+receivers:
+  # Receiver 1.
+  # <receiver type>:
+  examplereceiver:
+    # <setting one>: <value one>
+    endpoint: 1.2.3.4:8080
+    # ...
+  # Receiver 2.
+  # <receiver type>/<name>:
+  examplereceiver/settings:
+    # <setting one>: <value one>
+    disabled: false
+    # <setting two>: <value two>
+    endpoint: 127.0.0.1:9211
+```
 
-Its address can be configured in the YAML configuration file under section "receivers", subsection "opencensus" and field "address". The syntax of the field "address" is `[address|host]:<port-number>`.
+A receiver instance is referenced by its full name in other parts of the config,
+such as in pipelines. A full name consists of the receiver type, '/' and the
```

Comment:
IMO naming convention can be a suggestion but users are free to define any names for receivers as long as they are unique.

Reply:
@songy23 the name must start with receiver type otherwise it will not be recognised as a valid name. It be followed by a slash and a suffix.